test/with_api_v2/acceptance/cluster_test.go

```diff
@@ -28,6 +28,8 @@ func TestClusterDeduplication(t *testing.T) {
 	t.Parallel()
 
 	conf := `
+global:
```

Comment:
Is this to fix flaking tests? If so, let's move it to a separate PR and I will approve it 🙂 It sounds pedantic, but it makes it much easier to use `git blame`.

Reply:
I am certainly old school in bringing change sets to a GitHub pull request. Let's split the PR into two pieces then. The first to introduce snapshots and then the second to change the EndsAt behavior.

The reason to introduce the resolve timeout is from here:
https://github.com/prometheus/alertmanager/pull/3827/commits/2f0556893f5947c70371e188f9a3a893ffdbda69
```
Update the acceptance tests as the webhook now has a non-zero EndsAt
timestamp for firing alerts. The models.GettableAlert doesn't have a
`status` field which means the current test is unable to differentiate
between firing and resolved alerts.
```

Reply:
The PR to introduce the snapshot is https://github.com/prometheus/alertmanager/pull/3882. I will update this PR to include the behavior change for the webhook.